companion/flashing.md

```diff
@@ -16,23 +16,19 @@ To check power up the transmitter in Bootloader mode:
 ![bootloadersmall](https://user-images.githubusercontent.com/20209851/29190168-1b415e06-7de7-11e7-8d92-0010df929bbd.png)
 
 
-| Tx  | OpenTX Bootloader | PC OS | Add PC Driver |
+| Tx  | OpenTX Bootloader | Flash method |
 | --- | ------------------- | ------  | ----------------------- |
-| X7 | 2.2 | Any | No |
-| X9D | 1.x | Windows | Zadig |
-| X9D | 1.x | Linux | Dfu-util |
-| X9D | 1.x | Mac | Dfu-util |
-| X9D | 2.x | Any | No |
-| X9D+ | 2.x | Any | No |
-| X9E | 2.x | Any | No |
-| X12S | 2.2 + later | Windows | Zadig |
-| X12S | 2.2 + later| Linux | Dfu-util |
-| X12S | 2.2 + later| Mac | Dfu-util |
+| X7 | 2.2 |  Bootloader, Dfu-util |
+| X9D | 1.x | Dfu-util |
+| X9D | 2.x |  Bootloader, Dfu-util |
+| X9D+ | 2.x | Any |  Bootloader, Dfu-util |
+| X9E | 2.x | Any |  Bootloader, Dfu-util |
+| X12S | 2.2 + later | Windows | Dfu-util |
 
-MAC Note: Companion 2.2 for Mac comes with Dfu-util.
+Companion 2.2 for Mac and Windows comes with Dfu-util. Linux distribution usuall have a package for dfu-util. Windows needs drivers installed via Zadiag utility
 
 
-## Installing The Windows Driver If Required
+## Installing The Windows Driver with Zadiag (optional)
```

Comment:
Sure... I deliberately chose "If Required" based on forum feedback form people helping users who installed zadig and went through all of that when it was not required and did nothing to solve the problem they were addressing... but you approach is a different way... up to you...

Reply:
I would rather add an additional setence under that. if required in a title sounds a bit odd to me.